src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFolder2.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+using System.Linq;
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_StorageFolder2
+	{
+
+		readonly String[] _filenames = { "testfile1.txt", "testfile2.txt", "testfile3.txt", "testfile4.txt", "testfile5.txt" };
+		readonly String[] _foldernames = { "testfolder1", "testfolder2", "testfolder3", "testfolder4", "testfolder5.555" };
+		Windows.Storage.StorageFolder _folderForTestFiles;
+
+		[TestInitialize]
+		public async Task Init()
+		{
+			_folderForTestFiles = Windows.Storage.ApplicationData.Current.LocalFolder;

```

Comment:
Instead of putting the files directly in `LocalFolder`, please put them inside a subfolder (for example `StorageFolderGetItemsTest`). This way you can be sure there will be no conflicts with other tests, as some do write in `LocalFolder`

Reply:
ok.